keep/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "40ac95c0-35bd-49c8-a5f0-b21037bc87b4",
+  "app_id": "keep",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "An open-source AIOps Alert Management platform",
+    "title": "Keep",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Keep's providers page",
+        "image_url": "images/integrations-page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Keep's alerts single pane of glass",
+        "image_url": "images/alerts-page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Keep's noise-reduction rules builder page",
+        "image_url": "images/alert-rules.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Keep's workflows viewer",
+        "image_url": "images/workflows.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Keep's workflow builder page",
+        "image_url": "images/workflow-builder.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
```

Comment:
This should include all relevant fields as seen below as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags): 
      "Category::",
      "Offering::",
      "Queried Data Type::" If pulling data 
      "Submitted Data Type::",